Makefile

```diff
@@ -52,17 +52,22 @@ $(VIRTUALENV):
 	python3 -m venv $(PREFIX)
 	@echo ""
 
-install-git-scripts:
+LIBEXEC_DIR=$(DESTDIR)$(PREFIX)/libexec/apple-llvm
```

Comment:
This should use `:=` since this does not need to be lazily evaluated.  Also convention is spaces on either side.